src/lib/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { type MouseEventHandler, useEffect, useState } from 'react'
+
+import { InfoIcon } from 'lib/icons/Info.js'
+import { InfoDarkIcon } from 'lib/icons/InfoDark.js'
+
+interface TooltipProps {
+  children: React.ReactNode
+}
+
+export function Tooltip({ children }: TooltipProps): JSX.Element {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const handleClose = (): void => {
+    setIsOpen(false)
+  }
+
+  const handleEscape = (ev: KeyboardEvent): void => {
+    if (ev.key === 'Escape') {
+      handleClose()
+    }
+  }
+
+  useEffect(() => {
+    if (isOpen) {
+      document.addEventListener('click', handleClose)
+      window.addEventListener('keydown', handleEscape)
+    }
+
+    return () => {
+      document.removeEventListener('click', handleClose)
+      window.removeEventListener('keydown', handleEscape)
+    }
```

Comment:
Prefer `globalThis` over window (friendly to downstream tooling as `globalThis` is always defined.

```suggestion
    if (isOpen) {
      globalThis.document?.addEventListener('click', handleClose)
      globalThis.addEventListener?.('keydown', handleEscape)
    }

    return () => {
      globalThis.document?.removeEventListener('click', handleClose)
      globalThis.removeEventListener?.('keydown', handleEscape)
    }
```